workflows/prognostic_c48_run/runtime/steppers/machine_learning.py

```diff
@@ -198,7 +198,9 @@ def __call__(self, time, state):
             )
             tendency.update({"dQ2": dQ2_updated})
 
-        diagnostics["rank_updated_points"] = xr.where(dQ2_initial != dQ2_updated, 1, 0)
+        diagnostics["specific_humidity_limiter_active"] = xr.where(
```

Comment:
changed name of diagnostic to `specific_humidity_limiter_active` (which is valued either 0 or 1)